src/server/mod.rs

```diff
@@ -370,6 +370,7 @@ where
         let context = SourcingContext {
             t0: self.t0,
             scheduler: Rc::downgrade(&self.scheduler),
+            domain_probe: self.context.internal.input_probe().clone(),
```

Comment:
Let's settle on a name here. `input_probe` or `domain_probe`? Domain probe is probably more accurate, because not all inputs will be probed. Opinions? Let's pick one and use that consistently.

Reply:
I'll go with `domain_probe`